docs/pages/components/select.md

```diff
@@ -472,34 +497,37 @@ To add text in the body of the component, simply include your text in the `fd-li
 <br/>
 <br/>
 <br/>
-
+<label class="fd-form-label" id="b4546C40">
+    Error Select Example
+</label>
+<br>
 <div class="fd-popover">
    <div class="fd-popover__control">
        <div class="fd-select">
            <div class="fd-select__control is-error" role="button" tabindex="0" aria-controls="h07j9978H"  aria-expanded="false" aria-haspopup="true">
                Error
-               <span class="fd-button sap-icon--slim-arrow-down fd-select__button"></span>
+               <span class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></span>
            </div>
        </div>
    </div>
    <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown" aria-hidden="true" id="h07j9978H">
-        <ul class="fd-list fd-list--has-message fd-list--dropdown fd-list--compact" role="listbox">
-           <div aria-live="assertive" class="fd-list__message fd-list__message--error" role="alert">Error message</div>
-           <li class="fd-list__item is-selected" aria-selected="true" role="option" tabindex="0">
+        <div aria-live="assertive" class="fd-list__message fd-list__message--error" role="alert">Error message</div>
```

Comment:
Message is not visible anymore, it went behind select
![image](https://user-images.githubusercontent.com/26483208/83499476-ba530a80-a4bd-11ea-9ed9-65115bf3be75.png)

`.fd-dialog__body` is missing `position: relative`